app/_includes/md/plugins-hub/oidc-prereqs.md

```diff
@@ -22,15 +22,19 @@ difficulties during this phase, refer to the [Keycloak documentation](https://ww
    <img src="/assets/images/products/plugins/openid-connect/keycloak-client-service-auth.png">
    <br>
 {% if_plugin_version gte:3.5.x %}
-3. (Optional) Create another confidential client `cert-bound` with settings similar to the `service` client created previously.
+1. (Optional) Create another confidential client `cert-bound` with settings similar to the `service` client created previously.
    From the **Advanced** tab, enable the **OAuth 2.0 Mutual TLS Certificate Bound Access Tokens Enabled** toggle.
 {% endif_plugin_version %}
 {% if_plugin_version gte:3.6.x %}
-4. (Optional, to test mTLS Client Authentication) Create another confidential client `client-tls-auth` with settings similar to the `service` client created above. 
+1. (Optional, to test mTLS Client Authentication) Create another confidential client `client-tls-auth` with settings similar to the `service` client created above. 
    From the **Credentials** tab, select the **X509 Certificate** Client Authenticator and fill the Subject DN field so that it matches the Kong client certificate's, e.g.: `CN=JohnDoe, OU=IT`.
 {% endif_plugin_version %}
+{% if_plugin_version gte:3.7.x %}
+1. (Optional, to test Demonstrating Proof-of-Possession Client Authentication) Create another confidential client `client-dpop-auth` with settings similar to the `service` client created above. 
```

Comment:
```suggestion
1. **Optional**: To test Demonstrating Proof-of-Possession Client Authentication, create another confidential client `client-dpop-auth` with settings similar to the `service` client created above. 
```

Reply:
We have a line in this format before the change, so we need to keep consistent with it, or we need to change them altogether.